Lib/asyncio/runners.py

```diff
@@ -1,10 +1,17 @@
 __all__ = 'run',
 
+import signal
+
 from . import coroutines
 from . import events
 from . import tasks
 
 
+def sigint_handler(signum, frame):
+    """Interrupt the running loop when SIGINT is received"""
+    events.get_event_loop().interrupt()
```

Comment:
Hmmm. If `asyncio.run()` is launched from a non-main thread this code will interrupt the wrong loop.

Reply:
Thank you! I fixed this and added tests.